### PR TITLE
Fields for post entity added

### PR DESCRIPTION
**Pull request by orhungorkem**

### CODE REVIEW

File: practiceapp/src/main/java/com/practiceapp/practiceapp/entity/PostEntity.java [hunk 1]
```diff
@@ -17,6 +20,27 @@
...
+    private String text;
+
+    @Field
+    private Date date;
+
+    @Field
+    private String location;
+
+    @Field
+    private String author_id;
```
Comment (irem-zeynep) [inline practiceapp/src/main/java/com/practiceapp/practiceapp/entity/PostEntity.java:38]: In order to create a one-to-many relationship between the user and the posts, it should be written as `private UserEntity author`.

**Review by irem-zeynep — CHANGES_REQUESTED** [review]
The only problem is in the one-to-many relationship. Need to check.

File: practiceapp/src/main/java/com/practiceapp/practiceapp/entity/PostEntity.java (hunk 1 above)
Comment (orhungorkem) [reply practiceapp/src/main/java/com/practiceapp/practiceapp/entity/PostEntity.java]: I updated the relation. You may review again.

**Review by irem-zeynep — APPROVED** [review]
Perfect, thanks.

**Pull request merged**